administration/src/util/translations/de.json

```diff
@@ -20,11 +20,25 @@
     "wantsDigitalCard": "Ich beantrage eine digitale Ehrenamtskarte",
     "wantsPhysicalCard": "Ich beantrage eine physische Ehrenamtskarte",
     "blueCardJuleicaEntitlement": "Ich bin Inhaber:in einer JuLeiCa (Jugendleiter:in-Card)",
+    "blueCardWorkAtDepartmentEntitlement": "Ich bin aktiv in der Freiwilligen Feuerwehr mit abgeschlossener Truppmannausbildung bzw. abgeschlossenem Basis-Modul der Modularen Truppausbildung (MTA), oder im Katastrophenschutz oder im Rettungsdienst mit abgeschlossener Grundausbildung.",
```

Comment:
:upside_down_face: maybe we don't need the "." at the end of the entitlement descriptions.
or at least it should be the same for all items.
currently there is no "." in blueCardJuleicaEntitlement and goldenCardHonoredByMinisterPresidentEntitlement, but we have the "." for other items.

Reply:
well i think the requirements are entire sentences so they should have a dot when the user selects a requirement.
I think its not really needed to strip that. But you are right it should be equal for all requirements